src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -403,8 +403,16 @@ static constexpr Property<SchedulingCoreType> scheduling_core_type{"SCHEDULING_C
  * @brief This property allows CPU threads pinning during inference.
  * @ingroup ov_runtime_cpp_prop_api
  *
- * Developer can use this property to use or not use CPU threads pinning during inference. If user does not explicitly
- * set value for this property, OpenVINO may choose any desired value based on internal logic.
+ * Developer can use this property to enable or disable CPU thread pinning during inference on Windows and Linux. MacOS
+ * does not support CPU thread pinning, and this property is always disabled. If user does not explicitly set value for
```

Comment:
Suggest changes `CPU thread pinning` to `CPU threads pinning`.

Reply:
Update to CPU pinning as common definition